spec/acceptance/port_channel_spec.rb

```diff
@@ -37,7 +37,7 @@
     expect(result).to match(%r{flowcontrol_receive.*on}) if result =~ %r{flowcontrol_receive}
     expect(result).to match(%r{flowcontrol_send.*on}) if result =~ %r{flowcontrol_send}
     expect(result).to match(%r{speed.*10}) if result =~ %r{speed}
-    expect(result).to match(%r{mode.*passive}) if result =~ %r{mode}
+    expect(result).to match(%r{mode.*passive}) if result =~ %r{mode =>}
```

Comment:
Good catch that we are now matching the entire debug output.

Could we change speed and the 2 flowcontrol checks too to include the arrow?

Thanks!

Reply:
Updated all flow controls to include the `=>`